Makefile

```diff
@@ -34,21 +36,20 @@ serial_test:
 # A pattern rule that runs a single test script
 #
 $(tests): %.py :
-	export DSS_TEST_MODE=$${DSS_TEST_MODE:-standalone} \
-	&& coverage run -p --source=dss $*.py $(DSS_UNITTEST_OPTS)
+	coverage run -p --source=dss $*.py $(DSS_UNITTEST_OPTS)
 
 # Run standalone and integration tests
 #
 all_test:
-	DSS_TEST_MODE="standalone integration" $(MAKE) test
+	$(MAKE) test DSS_TEST_MODE="standalone integration"
```

Comment:
I would do `$(MAKE) DSS_TEST_MODE="standalone integration" test` but otherwise this change is fine by me.

Reply:
Fixed.